config/api.yml

```diff
@@ -986,18 +986,28 @@
     :options:
     - :collection
     - :subcollection
-    :verbs: *g
+    :verbs:
+    - :get
+    - :put
+    - :post
+    - :patch
```

Comment:
Minor but you should be able to use


```suggestion
    :verbs: *gppp
```

Reply:
Do we have that?  I don't see it in the list of sets

Reply:
Seems not, but then I'd prefer adding that one new set to keep the verbs section consistent.

Reply:
Done